wiki/Skin/id.md

```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+---
+
+# Skin
+
+*Lihat juga: [membuat skin](/wiki/Skinning)*\
+*untuk sub-forum khusus untuk skin, lihat [membuat skin](https://osu.ppy.sh/community/forums/15)*
```

Comment:
sub-forum names shouldn't be translated as they remain the same no matter what language you're currently browsing it on

![image](https://user-images.githubusercontent.com/36564236/107157600-88cd0000-69b7-11eb-9dba-eefe2987432d.png)

Reply:
Okeyy i already change it